siliconcompiler/report/dashboard/web/components/flowgraph.py

```diff
@@ -60,7 +60,7 @@ def get_nodes_and_edges(chip):
         node_color = NODE_COLORS[node_status]
 
         tool, task = get_tool_task(chip, step, index)
-        node_name = f'{step}{index}'
+        node_name = f'{step}/{index}'
```

Comment:
need to verify this works with the streamlit graphviz implementation

Reply:
Ahh, yes that is an excellent point. I forgot that the tool/task was already formatted that way